contracts/PartyBid.sol

```diff
@@ -183,7 +206,44 @@ contract PartyBid is Party {
             _ethFee = _closeSuccessfulParty(highestBid);
         }
         // set the contract status & emit result
-        emit Finalized(partyStatus, totalSpent, _ethFee, totalContributedToParty);
+        emit Finalized(partyStatus, totalSpent, _ethFee, totalContributedToParty, false);
+    }
+
+    // ======== External: Expire =========
+
+    /**
+     * @notice Determines whether a party can be expired. Any status other than `CanExpire` will
+     * fail the `expire()` call.
+     */
+    function canExpire() public view returns (ExpireCapability, string memory) {
```

Comment:
imo this fn would be cleaner if it just returned the enum. the strings follow our convention for errors right now which gives the wrong idea skimming the code